Navpage.html

```diff
@@ -0,0 +1,65 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+<!doctype html>
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+<link href="Navpagecss.css" rel="stylesheet" />
```

Comment:
I suggest renaming 'Navpagecss' to 'styles' because we will keep all the css code in 1 file, not only for navigation

Reply:
Good idea. I'll make sure to change it on future drafts since I'm not sure I can do it in this pull request